### DISCUSSION TITLE
ESP32APRS_T-TWR/Lilygo reboots randomly

### DISCUSSION THREAD

**Discussion by ve7ltn** [nakhonthai/ESP32APRS_T-TWR#31] [General]
Hi all, the ESP32APRS_T-TWR is really wonderful. I find that when I am monitoring using the web interface, the Uptime seems to reset to 0D 0:0 and when I watch the screen, I can see the welcome boot screen, and I see that it's rebooted. This happens sometimes in a few minutes, others 8-10 min. I've set the output power to low, server filter is set to M/25 to reduce the amount of traffic, and only messages is selected as far as RF2INET and INET2RF filters. I've only enabled radio and igate. Any suggestions? Thanks in advanced. 73

**Comment by Pedritoprint** [comment]
The same with me, The Device is randomly booting, not decoding any rf-Signals while good Signals audible and, because of that, no packets sending in RF2INET Statistics-Panel. I like to use it as I-Gate in HB but with this conditions not usable...sorry
73, HB9CMI

**Reply by ve7ltn** [reply]
Trying to figure out if it's the firmware or if it's the hardware.  I've tried with battery, or without battery. I've tried only running igate/radio, or digi/radio.  Stripped down the workload on it, can't find any common denominator.

**Reply by Pedritoprint** [reply]
Same for me. Reloaded x-times the firmware, using only igate/radio. Hardware is receiving APRS-Signals, tryed without squelch and with my old Kenwood TH D7E who has possibility for APRS, but no success. The Device with this problems isn't acceptable. I'm frustrated. BTW, The volume of the speaker in level "1" is to loud hihi.

**Reply by Pedritoprint** [reply]
I found a solution for not decoding the APRS-Signals over VHF. The Volume has to be on about Level 7, so i disconnected the speaker from Board and it works. But the thing with reboot?? still exists.

**Reply by ve7ltn** [reply]
I haven't pulled that connector yet, but I'm wondering if it's a hardware problem, or if it's the firmware.  Since mine doesn't stay alive long enough, I can't test to see if the igate is even doing what it's supposed to.  I tried sending messages but don't get the ACK, which to me means the igates around me are RX only.  This was supposed to solve that, but I'm not getting the ACK, so my messages get sent 6 times then more less die on the vine.

**Reply by podgorskik** [reply]
I have an observation and HW modification proposal. Check please issue #45 
Regards
Krzysztof SP9SVH
